tests/secrets_ospdo.yaml

```diff
@@ -0,0 +1,97 @@
+# oc header override
+oc_header: ''
+
+install_yamls_path: /root/src/github.com/install_yamls
+
+dpa_tests_dir: /root/data-plane-adoption_pkomarov/tests
```

Comment:
the var "dpa_tests_dir" would go away ,
but there are some variables that need to be present because of how ospdo is structured differently , 
for instance the "openstack" namespace is currently a must for ospdo , so that the rhoso must be different : 

That said , I can put this in the CI side of things so the specifics are seperated from the adoption logic

`osp18_namespace: osp18

ospdo_namespace: openstack

osp18_ns_suffix: "-{{ osp18_namespace }}"

mysql_client_override: >-
  --overrides='{
    "apiVersion": "v1",
    "metadata": {
        "annotations": {
            "k8s.v1.cni.cncf.io/networks": "[{\"name\": \"internalapi-static\",\"namespace\": \"openstack\", \"ips\":[\"172.17.0.99/24\"]}]"
        }
    },
    "spec": {
        "nodeName": "'"{{ controller_node.stdout }}"'",`

Reply:
I think the new variables themselves are fine (as added into `tests/vars.sample.yaml`). What i meant was that the `tests/secrets_ospdo.yaml` file should probably not be included in this repo at all, instead it should live in the CI configuration. I now realize my comment was confusing in that regard, sorry :).

Reply:
got it , agreed , I'll make move the secrets_ospdo.yaml to the CI side and leave the general vars in vars.sample.yaml